acceptance-test/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
     ext {
         dropwizardVersion = '3.1.3'
         curatorVersion = '4.2.0'
```

Comment:
for note: the upgrade to 5.0 is not planned due to braking changes https://curator.apache.org/breaking-changes.html. the upgrade of curator library will be made as separate task.